beach UUID changed to long id

Moburger opened the pull request:
Check BeachService row 30, correct?

Moburger committed:
beach UUID changed to long id
Check BeachService row 30, correct?

eric-muhire reviewed (changes requested):
row 30 should be:
beach.setId(request.getId());

Moburger committed:
beach UUID changed to long id
Check BeachService row 30, correct?

changed: added private long id; to add beach request
and changed getbyid

eric-muhire reviewed (approved):
Have you run the code  if so and it is working ok to merge
